static-build/qemu-virtiofs/Dockerfile

```diff
@@ -38,6 +38,8 @@ RUN apt-get install -y \
 
 RUN cd  .. && git clone "${QEMU_VIRTIOFS_REPO}" qemu-virtiofs
 RUN git checkout "${QEMU_VIRTIOFS_TAG}"
+ADD qemu/patches/virtiofsd/0001-add-time-to-seccomp.patch /root/0001-add-time-to-seccomp.patch
```

Comment:
You could add the complete dir and then iterate but I hope we dont need more patches as this will be quickly added to virtiofsd repo.